eclipse-collections/src/main/java/org/eclipse/collections/impl/multimap/bag/sorted/mutable/SynchronizedSortedBagMultimap.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (c) 2016 Shotaro Sano.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.impl.multimap.bag.sorted.mutable;
+
+import java.io.Serializable;
+import java.util.Comparator;
+
+import net.jcip.annotations.GuardedBy;
+import org.eclipse.collections.api.bag.sorted.MutableSortedBag;
+import org.eclipse.collections.api.block.function.Function;
+import org.eclipse.collections.api.block.function.Function2;
+import org.eclipse.collections.api.block.predicate.Predicate2;
+import org.eclipse.collections.api.multimap.Multimap;
+import org.eclipse.collections.api.multimap.bag.MutableBagMultimap;
+import org.eclipse.collections.api.multimap.list.MutableListMultimap;
+import org.eclipse.collections.api.multimap.sortedbag.ImmutableSortedBagMultimap;
+import org.eclipse.collections.api.multimap.sortedbag.MutableSortedBagMultimap;
+import org.eclipse.collections.api.tuple.Pair;
+import org.eclipse.collections.impl.factory.Multimaps;
+import org.eclipse.collections.impl.multimap.AbstractSynchronizedMultimap;
+
+public class SynchronizedSortedBagMultimap<K, V>
+        extends AbstractSynchronizedMultimap<K, V>
+        implements MutableSortedBagMultimap<K, V>, Serializable
+{
+    private static final long serialVersionUID = 1L;
+
+    public SynchronizedSortedBagMultimap(MutableSortedBagMultimap<K, V> newMultimap)
+    {
+        super(newMultimap);
+    }
+
+    public SynchronizedSortedBagMultimap(MutableSortedBagMultimap<K, V> newMultimap, Object newLock)
+    {
+        super(newMultimap, newLock);
+    }
+
+    /**
+     * This method will take a Multimap and wrap it directly in a SynchronizedMutableMultimap.
```

Comment:
I think we should not mention _wrap_ here as a new MutableMultimap will be created.